Core/Src/Wireless/Wireless.c

```diff
@@ -163,3 +165,25 @@ void Wireless_DMA_Handler(SX1280* SX){
     DMA_Callback(SX);
     SX->expect_packet = false;
 }
+
+void updateChannel(WIRELESS_CHANNEL channel) {
+    SX1280* SX_transmit = &SX1280_TX_struct;
+    SX1280* SX_receive = &SX1280_RX_struct;
+
+    switch (channel) {
+        case YELLOW_CHANNEL: {
+            setChannel(SX_transmit, WIRELESS_YELLOW_COMMAND_CHANNEL);
+            setChannel(SX_receive, WIRELESS_YELLOW_FEEDBACK_CHANNEL);
```

Comment:
The receive channel might require a call to "setRX(SX_RX, SX_RX->SX_settings->periodBase, 0xFFFF);" after setting the frequency to go back to the receiving state

Reply:
I tested this code, and it did work correctly. Does that mean I should still change it?

Reply:
If it works it works. I didn't know if it was needed or not.